ckan/Dockerfile

```diff
@@ -15,8 +15,8 @@ RUN  pip3 install -e git+https://github.com/GenomicDataInfrastructure/gdi-userpo
 
```

Comment:
Is this  file also not obsolete now, because this project is focussing on local development?

Reply:
Yes and no, I would say. 

In theory, the development life cycle includes code, test, build, publish. The artefact delivered by this repo is a docker image. Any suggestion on how to blend the two docker files together?

Reply:
Peharps, to not block this PR, I suggest we open another issue to review this later. What do you think, @hcvdwerf ?